Remove canvas minimap when viewing fns

StachuDotNet opened the pull request:
Will resolve #4006.

pbiggar commented:
Gave it a quick look, and directionally this looks right.

StachuDotNet commented on client/styles/_canvas.scss:
```diff
@@ -230,11 +230,10 @@ body #app * {
+    a {
+      width: 110px;
       height: auto;
+      bottom: 10px;
```
does nothing; remember how to CSS

StachuDotNet commented on client/styles/_canvas.scss:
```diff
@@ -208,16 +208,16 @@ body #app * {
   text-align: center;
   color: $cyan;
-  font-size: 10px;
+  font-size: 14px;
```
todo: figure out a standard font size, or more likely use the same font as the docs icon next to it

pbiggar commented on client/src/app/Main.res:
```diff
@@ -1975,7 +1970,7 @@ let update_ = (msg: msg, m: model): modification => {
     Many(list{
       ReplaceAllModificationsWithThisOne(
-        m => ({...m, canvasProps: {...m.canvasProps, minimap: None}}, Cmd.none),
+        m => ({...m, canvasProps: m.canvasProps,}, Cmd.none),
```
```suggestion
        m => (m, Cmd.none),
```

StachuDotNet replied:
Ah, good point.

That leaves the code like this:
```
| GoToArchitecturalView =>
    Many(list{
      ReplaceAllModificationsWithThisOne(
        m => (m, Cmd.none),
      ),
      Deselect,
      MakeCmd(Url.navigateTo(Architecture)),
    })
```

It seems like I can remove the `ReplaceAllModificationsWithThisOne` element of that list, then?

StachuDotNet replied:
Maybe this is wrong - could be some sort of 'reset pending modifications' that I don't understand.

- [ ] figure out what `ReplaceAllModificationsWithThisOne` is - it seems a bit scattered in code.

StachuDotNet replied:
after more inspection, I'm going to remove it - seems useless.

pbiggar commented on client/styles/_canvas.scss:
```diff
@@ -207,34 +207,31 @@ body #app * {
 
 .minimap {
```
Is there a reason for the minimal class to exist anymore? If so, it should be renamed to something else?

pbiggar reviewed (changes requested):
Looks good but I don't understand the CSS changes. Can you go though them (or remove the css if it's not needed anymore)?

StachuDotNet commented:
Renamed `minimap` to `back-to-canvas`.

The remaining styling, with commentary:
```
.back-to-canvas {
  // positioning
  position: fixed;
  bottom: 4px;
  right: 65px;
  width: 170px;

  // styling to be consistent with the UI elements and surrounding features (docs link)
  cursor: pointer;
  text-align: center;
  color: $cyan;
  font-size: 16px;

  .back-to-canvas-content {
    // for some centering of the text and icon
    display: flex;
    align-items: center;
    justify-content: space-between;

    .help-icon {
      font-size: 45px;
      color: $cyan;
    }
    a {
      width: 110px;
    }
  }

  .tooltips {
    // otherwise, the tooltip is too far to the right of the ?
    right: 23px;
  }
}
```

StachuDotNet commented:
I just experienced this lag in prod, finally - wow! bad.
